packages/cadence-parser/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "@onflow/cadence-parser",
```

Comment:
[@onflow/cadence-parser](https://www.npmjs.com/package/@onflow/cadence-parser) is an existing package, we will need a different name to avoid this conflict.

Reply:
Good catch @jribbink thanks